.joss/paper.md

```diff
@@ -0,0 +1,90 @@
+fMRI data follows a relatively stereotyped shape from the Blood Oxygen Level Dependent (BOLD) response, which is best modelled with a double-gamma function (e.g. not a gaussian/normal function).
+The peak of the gamma curve (i.e. activation/deactivation) is determined by the beta coefficient.
+Thus a larger betas mean greater activation, and smaller or negative betas mean less activation or deactivation.
+Traditional fMRI analysis will group together all the relevant trial types and give them all one beta estimate, where variance between trials is treated as noise.
+NiBetaSeries, on the other hand, gives each trial its own beta estimate treating the variance between trials as the signal of interest.
+The final complication is the multiple methods we can use to derive individual beta estimates.
+Two common methods are least squares all (LSA) and least squares separate (LSS).
+LSA places all the trials in the same GLM, where each trial is a separate predictor.
+LSA works well when the trials are far apart in time since the BOLD response take a long time to return to baseline.
+When the trials are close together, however, the bold responses start to overlap and the GLM cannot accurately attribute the variance in the fMRI data to trial 1 or trial 2, leading to unreliable beta estimates.
+LSS tackles this problem by making as many GLMs as there are trials.
+For each trial, a GLM is created where one predictor is the trial of interest and the other predictor is the combination of the other trials.
+This reduces the amount of overlap (or more accurately correlation) between predictors, leading to more reliable individual beta estimates.
+NiBetaSeries currently implements LSS making it an ideal candidate for experiments with trials that occur closer together (e.g. 3-7 seconds apart on average).
+
+The second question of what an atlas parcellation is will hopefully be easier to explain.
+The brain can be cut up in many ways, either via anatomical landmarks, by which regions are activated by a task of theoretical interest, by how the brain organizes itself when not engaged in any external task, or a number of other creative methods.
+However the brain is sliced up, we need to mark all the selected voxels within a brain region with the same number to uniquely identify the region.
+A voxel is shortened from the term volumetric pixel, unlike typical pictures our brain images are 3 dimensional, so in order to account for the volume we use 3D voxels instead of 2D pixels.
+An image with all the regions uniquely labelled is considered an atlas parcellation for the purposes of NiBetaSeries.
+The unique regions will be used to average across the voxels containing beta estimates for each trial.
+The averaging of beta estimates is repeated over all the trials resulting in a list of averaged beta estimates for that region.
```

Comment:
unsure at face value what the average is of that is repeated, averaging of beta estimates across each voxel in the parcel?